apps/web/.env.example

```
@@ -29,6 +29,7 @@ UPSTASH_REDIS_REST_TOKEN=
 
 # Upstash QStash – required for queues and background jobs
 # Get your QStash Token here: https://upstash.com/docs/qstash/overall/getstarted
+QSTASH_URL="https://qstash-us-east-1.upstash.io"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)